### PR TITLE
Issue 14

### PR DESCRIPTION
**Pull request by KD335**
flask_app/flask_main.py crashes after splitting the data, but everything else appears to work.

### CODE REVIEW

File: flask_app/api/map_paths.py [hunk 1]
```diff
@@ -11,7 +11,7 @@ def correct_action(current_state):
    elif current_state == 'prepare':
       actions = ['actions/split_data.html']
    elif current_state == 'split':
-      actions = ['actions/train_model.html']
+      actions = ['actions/train_lin_reg.html']
```
Comment (john-coffman) [inline flask_app/api/map_paths.py:14]: Be careful renaming variables without changing the name of the file that it is accessing. This is what caused the `error TemplateNotFound: actions/train_lin_reg.html` when going from split to train.

File: pandas_code/code_templates/eval_lin_reg.py [hunk 2]
```diff
@@ -0,0 +1,8 @@
+# evaluate the effectiveness of the model
+from sklearn.metrics import mean_squared_error
+def get_code(args):
+   train_error = mean_squared_error(args[0], args[2])
+   test_error = mean_squared_error(args[1], args[3])
+   error_change = abs(test_error - train_error)
+   return error_change
+   # ^ don't really need to return but pylint complained
```
[the remark this answers is not in the document]
Comment (KD335) [reply pandas_code/code_templates/eval_lin_reg.py:8]: I thought we were not allowed to change pylint

File: flask_app/api/map_paths.py (hunk 1 above)
Comment (KD335) [reply flask_app/api/map_paths.py:14]: I didn't add anything to the flask app that would create a page for it I simply renamed train_model to train_lin_reg everywhere it appeared in the code itself so I'm guessing the class name is still train_model.html, but I can change that real quick. As it stands though, I don't believe there is anything connecting flask to the model evaluation so if the code is generated through the flask app, then the evaluation won't be added.

**Review by john-coffman — APPROVED** [review]
Everything looks good.

**Pull request merged**